doc/uninstalling.md

```diff
@@ -1,8 +1,9 @@
 # Uninstalling the Operator
 
-If for some reason the operator needs to be uninstalled/deleted, there are still two ways in which Spinnaker itself can be prevented from being deleted, explained in the following sections.
+If Operator needs to be uninstalled or deleted, you can keep the Spinnaker deployments from being deleted in one of two ways: [replace Operator with Halyard](#replacing-the-operator-with-halyard) or [remove Operator ownership of Spinnaker resources](#removing-operator-ownership-of-spinnaker-resources).
```

Comment:
Is the link `#replacing-the-operator-with-halyard` still correct? I see that the name of the section was slightly changed below to `Replacing Operator with Halyard`

Reply:
Updated the verbs to match